go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/doresoap/ytools
```

Comment:
Looks like I found a typo :-) I guess this should be `github.com/codesoap/ytools`.